api/compute/v1alpha1/common.go

```diff
@@ -390,21 +390,24 @@ const (
 )
 
 func validResourceRequirement(requirements corev1.ResourceRequirements) bool {
+	if requirements.Limits.Cpu().Sign() == 0 {
+		return validResource(requirements.Requests) && validResource(requirements.Limits) &&
+			requirements.Requests.Memory().Cmp(*requirements.Limits.Memory()) <= 0
+	}
 	return validResource(requirements.Requests) && validResource(requirements.Limits) &&
 		requirements.Requests.Memory().Cmp(*requirements.Limits.Memory()) <= 0 &&
 		requirements.Requests.Cpu().Cmp(*requirements.Limits.Cpu()) <= 0
 }
 
 func validResource(resources corev1.ResourceList) bool {
-	// cpu & memory > 0 and storage >= 0
-	return resources.Cpu().Sign() == 1 &&
+	// memory > 0 and cpu & storage >= 0
+	return resources.Cpu().Sign() >= 0 &&
```

Comment:
will it be okay for `requests.cpu` to be 0?

Reply:
Make sense, I will set it to the default value if `requests.cpu` is 0